sqlglot/schema.py

```diff
@@ -308,6 +308,22 @@ def get_column_type(
 
         return exp.DataType.build("unknown")
 
+    def has_column(
+        self,
+        table: exp.Table | str,
```

Comment:
what if there's a catalog and db?

Reply:
they will be included in the AST node for the table - either the supplied one or the one that will be parsed from the `str` value